src/lib/inputs/Input.svelte

```diff
@@ -89,9 +93,13 @@
 <div class={containerClassString}>
   {#if label}
     {#if labelIn}
-      <label class="label-in" for={$$restProps.id}>{$$restProps.id}</label>
+      <label class="label-in" for={$$restProps.id} style="color: {labelColor}"
+        >{$$restProps.id}</label
+      >
     {:else}
-      <label class="label" for={$$restProps.id}>{$$restProps.id}</label>
+      <label class="label" for={$$restProps.id} style="color: {labelColor}"
+        >{$$restProps.id}</label
+      >
     {/if}
   {/if}
   <input
```

Comment:
have you considered exposing other 'standard' html attributes on the interface for this one like step, min, max etc https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/number#additional_attributes 
for a component library i'd prob pass on the 'list' but the others may be of interest

Reply:
nevermind lol